docs/Manager.md

```diff
@@ -1,5 +1,40 @@
 #Phidget Manager
 
-[Basic Manager example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/manager.js)
+The Phidget Manager library makes for an intuitive and easy way to manage your Phigets devices. For a quick start into your Manager project see the [Getting Started](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Manager.md#GettingStarted) section or jump to this [basic Manager example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/manager.js).
```

Comment:
In the future we should consider creating a simpler version of what you linked to and having it in our documentation.